cfg/cfg.go

```diff
@@ -126,12 +121,6 @@ func init() {
 		"",
 		"Default address of registry from which to retrieve meta.yamls when plugin FQNs do not specify a registry. Ignored unless --download-metas is set",
 	)
-	flag.BoolVar(
-		&DownloadMetas,
-		"download-metas",
-		false,
-		"Download plugin metadata from registry instead of process already-downloaded metas",
-	)
```

Comment:
Removing this now would result in a minor version bump (0.16.x) and require changes in upstream Che.

Reply:
Yes, but I'm considering change version to `v1.0.0` since it is backward incompatible and first number is supposed be reflect such changes. WDYT?

Reply:
I'm open to either option but worry we're not to the point of calling it a `v1.0` release, especially with the various naming convention/registry changes still coming down the pipe. It's still okay to break backward compatibility in version zero so I'm leaning more in that direction.